scripts/e2e-cluster-dump.sh

```diff
@@ -79,96 +79,116 @@ function cluster-describe {
     kubectl -n mayastor describe daemonsets
 }
 
-function logs-csi-containers {
-    mayastor_csipods=$(kubectl -n mayastor get pods | grep mayastor-csi | sed -e 's/ .*//')
-    for pod in $mayastor_csipods
-    do
-        echo "# $pod csi-driver-registrar $* ---------------------------------"
-        kubectl -n mayastor logs "$@" "$pod" csi-driver-registrar
-    done
+function hasPodRestarted {
+    rst=$(kubectl -n mayastor get pods "$1" | grep -v NAME | awk '{print $4}')
+    return $((rst))
+}
 
-    moacpod=$(kubectl -n mayastor get pods | grep moac | sed -e 's/ .*//')
-    echo "# $moacpod csi-provisioner $* ---------------------------------"
-    kubectl -n mayastor logs "$@" "$moacpod" csi-provisioner
-    echo "# $moacpod csi-attacher $* ---------------------------------"
-    kubectl -n mayastor logs "$@" "$moacpod" csi-attacher
+# args filename kubectlargs
+# filename == "" -> stdout
+function kubectlEmitLogs {
+    fname=$1
+    shift
+
+    if [ -n "$fname" ]; then
+        kubectl -n mayastor logs "$@" >& "$fname"
+    else
+        kubectl -n mayastor logs "$@"
+    fi
 }
 
-function logs-csi-mayastor {
+# args = destdir podname containername
+# if $destdir != "" then a log files are generate in $destdir
```

Comment:
should be "then log files are generated"

Reply:
Done